core-web/libs/block-editor/src/lib/extensions/bubble-form/bubble-form.extension.ts

```diff
@@ -49,15 +58,17 @@ export const BubbleFormExtension = (viewContainerRef: ViewContainerRef) => {
         addCommands() {
             return {
                 openForm:
-                    () =>
+                    (form, options) =>
                     ({ chain }) => {
-                        return chain()
+                        chain()
                             .command(({ tr }) => {
-                                tr.setMeta(BUBBLE_FORM_PLUGIN_KEY, { open: true });
+                                tr.setMeta(BUBBLE_FORM_PLUGIN_KEY, { form, options, open: true });
 
                                 return true;
                             })
                             .run();
+
+                        return formValue$;
```

Comment:
I'm still a little worried about this. Should we go through the block editor test cases to see that this doesn't break anything?